source/user.scripts/usr/local/emhttp/plugins/user.scripts/Userscripts.page

```diff
@@ -90,8 +90,16 @@ foreach ($schedule as $scriptSchedule) {
 }
 $scheduleScript .= "</script>";
 ?>
+<script src="/plugins/compose.manager/javascript/ace/ace.js" type= "text/javascript"></script>
 <script>
 var caURL = "/plugins/user.scripts/exec.php";
+var aceTheme=<?php echo (in_array($theme,['black','gray']) ? json_encode('ace/theme/tomorrow_night') : json_encode('ace/theme/tomorrow')); ?>;
```

Comment:
These dark and light themes are the ones that bonienl chose for the file manager plugin